scripts/privacyidea

```diff
@@ -150,10 +149,8 @@ def yubi_mass_enroll(lotpc,
 
         elif yubi_mode == MODE_YUBICO:
             yubi_otplen = 32
-            if yubi_prefix_serial:
-                yubi_otplen = 32 + len(serial) * 2
-            elif yubi_prefix:
-                yubi_otplen = 32 + (len(yubi_prefix) * 2)
+            if prefix:
+                yubi_otplen = 32 + len(prefix) * 2
```

Comment:
the 'prefix' returned by 'enrollYubikey' is already Hex-coded and thus we don't need to duplicate the prefix length.

Reply:
will fix this.